libexec/alluxio-config.sh

```diff
@@ -90,20 +90,28 @@ ALLUXIO_SERVER_CLASSPATH="${ALLUXIO_CONF_DIR}/:${ALLUXIO_CLASSPATH}:${ALLUXIO_AS
 ####################################################################################################
 function getConf {
   "${JAVA}" -cp ${ALLUXIO_CLIENT_CLASSPATH} ${ALLUXIO_JAVA_OPTS} -Dalluxio.logger.type=Null \
+      -Dalluxio.master.audit.logger.type=Null \
```

Comment:
what's the default value for `alluxio.master.audit.logger.type`? why do we want to set it to Null here?